scripts/apps/desks/views/main.html

```diff
@@ -1,40 +0,0 @@
-<div class="subnav subnav--absolute">
-    <div class="button-list button-list--padded desk-tabs">
-        <a href="" ng-repeat="v in views" ng-class="{'toggle-button--active': v === view}" id="{{v}}" ng-click="setView(v)" class="toggle-button">{{v | translate}}</a>
-        <div class="online-users" ng-if="view === 'users'">
-            <p><span sd-check ng-model="online_users" name="online_users" id="online_users" ng-change="changeOnlineUsers(online_users)"></span></p>
-            <label translate>show only online users</label>
-        </div>
-    </div>
-</div>
-
-<section class="main-section master-desk-view" sd-desk-config>
-
-    <div class="main-desk-container">
-
-        <div class="desk-container" ng-if="view === 'content'" ng-repeat="desk in desks._items | orderBy: 'name'">
-            <div class="desk" ng-include="'scripts/apps/desks/views/desk.html'"></div>
-        </div>
-
-        <div class="desk-container" ng-if="view === 'tasks'" ng-repeat="desk in desks._items | orderBy: 'name'">
-            <div class="desk" ng-include="'scripts/apps/desks/views/desk-tasks.html'"></div>
-        </div>
-
-        <div class="desk-container" ng-if="view === 'users'" ng-repeat="desk in desks._items | orderBy: 'name'">
-            <div class="desk" ng-include="'scripts/apps/desks/views/desk_users.html'"></div>
-        </div>
-        <div class="desk-container" ng-if="view === 'sluglines'" ng-repeat="desk in desks._items | orderBy: 'name'">
-            <div class="desk" ng-include="'scripts/apps/desks/views/slugline.html'"></div>
```

Comment:
You should also check these included views(and their child views) and remove them too if they are not used anywhere else.

Reply:
I see you have deleted these views, but not their dependencies that are not used anywhere else.

You should delete it recursively - open the file that you want to delete and see if it depends on something else, a directive for example. See if that directive is used somewhere else - if not mark it for deletion, but also look at the dependencies of that directive and so on until you get to a leaf node in the dependency tree. It may sound like a lot of work, but it will probably be only a few extra files in reality.

Reply:
Looks like you forgot `sd-task-status-items` in `/scripts/apps/desks/views/desk-tasks.html`.

Reply:
done 👍